### PR TITLE
feat: add stop and vehicle actions

### LINKED ISSUE #131: cross app links
We want cross app links to: 
* nah.sh
* future: divers scooter apps

### PR DESCRIPTION
**Pull request by anbraten**
closes #131

Screenshot from 2024-08-21 18-20-04

image

### CODE REVIEW

File: app/src/components/popups/Actions.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <div v-if="actions.length > 0 && checkFeatureFlag('vehicle_stop_actions').value" class="flex mt-2 overflow-x-auto">
+    <div class="inline-flex w-min gap-2 pb-4">
+      <template v-for="action in actions" :key="action.url">
+        <a :href="action.url" target="_blank" rel="noopener noreferrer" class="flex flex-shrink-0">
+          <button
```
Comment (lukashass) [inline app/src/components/popups/Actions.vue:6]: a and button should not be nested

**Review by lukashass — CHANGES_REQUESTED** [review]
@anbraten why are the action buttons blue and not red?

File: app/src/components/popups/Actions.vue (hunk 1 above)
Comment (anbraten) [reply app/src/components/popups/Actions.vue]: changed code to use our existing button

**Comment by anbraten** [comment]
image

image

**Review by lukashass — APPROVED** [review]

**Pull request merged**